corc-cascading/src/test/java/com/hotels/corc/cascading/SearchArgumentFactoryTest.java

```diff
@@ -44,20 +48,34 @@
 
   private SearchArgumentFactory.Builder builder;
 
+  //Don't know how to get the type right for this setup
   @Before
   public void setup() {
     builder = new Builder(mockInternal);
     when(mockInternal.startNot()).thenReturn(mockInternal);
-    when(mockInternal.lessThan(anyString(), any())).thenReturn(mockInternal);
-    when(mockInternal.lessThanEquals(anyString(), any())).thenReturn(mockInternal);
+    when(mockInternal.lessThan(anyString(), same(toType(ONE)), any())).thenReturn(mockInternal);
```

Comment:
Instead of `toType(ONE)` can you just use `PredicateLeaf.Type.LONG`? If so you could declare this type at the top as something like `ONE_TYPE` and then use this everywhere instead and then remove the `toType` method in this class.